### ISSUE TITLE
App freezes / hard locks up device.

### ISSUE THREAD

**Issue by BugReportering** [SkyTubeTeam/SkyTube#933]
Just loading the app without even picking a video causes this issue much of the time.

**Comment by gzsombor** [comment]
Unable to reproduce without details.

**Comment by BugReportering** [comment]
Let me know which details  you would like me to provide and I will attempt to do so to the best of my ability. 
The device used is my S3 (with Android 4.1.2).

**Comment by BugReportering** [comment]
130|shell@android:/ $ logcat | grep SkyTube
E/AndroidRuntime(24089):        at free.rm.skytube.app.SkyTubeApp.notifyUserOnError(SkyTubeApp.java:281)
<hard lockup>

**Comment by kdsdata** [comment]
Happened to me a couple of days ago  with free.rm.skytube.oss_37.apk, total lock up, but I can't say if it was the App or the Android.

**Comment by gzsombor** [comment]
I saw a couple of freezes - for around a 20 secs - with the latest release, when moving from the channel browser to back to the subscription tab, which caused by - I think - too many parallel database query, when videos were collected from subscriptions, as I have ~120 channels subscribed with a couple of years of metadata about the videos. I could fix it that - or at least, with the latest builds, I can't see that problem anymore. I haven't encountered total lockup as of yet. Do you have lot's of subscribed channel with huge list of videos?

**Comment by BugReportering** [comment]
I had zero subscriptions.

**Comment by kdsdata** [comment]
No, I don't have any subscriptions.  You mentioned around 20 seconds.  I can't say for sure if I waited that long after what seemed a freeze.  If so, it may then be related to my bug report [](https://github.com/SkyTubeTeam/SkyTube/issues/940), where there is a long delay after a sideswipe, and yes, they can be as long as 20 seconds.  That delay is not the repositioning to the position in the video, that comes up relatively fast, but it is delay in the restart of the play at that position.  I'll add this comment to my bug report.
BTW, I want to say thanks to the whole team.  SkyTube is such a user friendly piece of work.  As a minimalist (small projects) programmer I really appreciate not only the original effort, but the ongoing effort. It's AWESOME, THANKS, kdsdata.

**Comment by gzsombor** [comment]
Thanks @kdsdata for the kind words, it's rare that man can read positive opinions in this issue tracker :) Which is somehow expected, because if everything is awesome, you wont open a bug ticket :smile: 
 I hope F-Droid could build the new version in the near future, so you can re-test the issue, if you still see the problem (or with downloading SkyTube Extra from github).

**Comment by kdsdata** [comment]
Please see my related comment under [](https://github.com/SkyTubeTeam/SkyTube/issues/940#issuecomment-806243811)
